scripts/globals/confrontation.lua

```diff
@@ -0,0 +1,58 @@
+require("scripts/globals/common")
+require("scripts/globals/npc_util")
+require("scripts/globals/settings")
+require("scripts/globals/status")
+require("scripts/globals/zone")
+-----------------------------------
+
+tpz = tpz or {}
+tpz.confrontation = tpz.confrontation or {}
+
+tpz.confrontation.start = function(player, npc, mobId)
```

Comment:
the heck is all this?? I was able to tie it all to player and mob in the core the same way bcnm is tied. was much shorter  because I was able to reuse existing code.

Reply:
yo dude i didn't mean to have you delete all your work - I dunno what you mean to do wid it yet